src/django/api/models/__init__.py

```diff
@@ -68,6 +68,7 @@
 from .production_type import ProductionType
 from .request_log import RequestLog
 from .sector import Sector
+from .sector_group import SectorGroup
 from .source import Source
```

Comment:
Let's not add Models to __init__.py to avoid disambiguates

Reply:
Added import of SectorGroup to the admin.py